src/assets/translations/es.translations.json

```diff
@@ -36,9 +36,9 @@
   "modal.text.7": " Sí",
   "modal.text.8": " No",
   "modal.text.9": "Selecciona tu tipo de dieta",
-  "modal.text.10": "Menú normal",
```

Comment:
This is wrong i don't want to change this

Reply:
I changed the values already